package.json

```diff
@@ -29,26 +29,26 @@
   },
   "homepage": "https://github.com/web-push-libs/web-push#readme",
   "dependencies": {
-    "asn1.js": "^4.8.1",
+    "asn1.js": "^5.0.0",
     "http_ece": "^0.5.2",
     "jws": "^3.1.3",
     "minimist": "^1.2.0",
     "urlsafe-base64": "^1.0.0"
   },
   "devDependencies": {
-    "chalk": "^1.1.3",
-    "chromedriver": "2.24.1",
-    "del": "^2.2.1",
-    "eslint": "^3.5.0",
-    "eslint-config-airbnb": "^11.1.0",
-    "eslint-plugin-import": "^1.16.0",
-    "geckodriver": "1.1.3",
+    "chalk": "^2.3.0",
+    "chromedriver": "^2.33.2",
+    "del": "^3.0.0",
+    "eslint": "^4.12.1",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-plugin-import": "^2.8.0",
+    "geckodriver": "^1.10.0",
     "istanbul": "^0.4.2",
     "mkdirp": "^0.5.1",
-    "mocha": "^3.0.2",
+    "mocha": "^4.0.1",
     "portfinder": "^1.0.2",
-    "selenium-assistant": "1.0.0",
-    "selenium-webdriver": "3.0.0-beta-2",
+    "selenium-assistant": "^5.2.0",
+    "selenium-webdriver": "^3.6.0",
```

Comment:
I'm not 100% sure if this is needed. Selenium-assistant is pretty much built to work with a specific selenium-webdriver instance it defines.

https://github.com/GoogleChromeLabs/selenium-assistant/blob/master/package.json#L57

However, that will not help with the Node 4 travis issue,